src/styles/base.scss

```diff
@@ -73,6 +73,11 @@ img.fail {
   visibility: hidden;
 }
 
+.abbreviation {
```

Comment:
Please, don't define global styles. Instead, create a component or mixin or at least import a common css file where it is needed.